src/main/scala/NumberNotations.scala

```diff
@@ -0,0 +1,69 @@
+
+object NumberNotations {
+
+  val englandShort = List("sextillion",
+    "quintillion",
+    "quadrillion",
+    "trillion",
+    "billion",
+    "million",
+    "thousand")
+
+  val european = List("trilliard",
+    "trillion",
+    "billiard",
+    "billion",
+    "milliard",
+    "million",
+    "thousand")
+
+  def notationSplit(num: String): List[String] = {
+    num.reverse.grouped(3).toList.map(_.reverse.toList.mkString("")).reverse
+  }
+
+  def stringBuild(list: List[String]): String = {
+    def buildRecursively(list: List[String], result: String = ""): String = {
+      list.headOption match {
+        case Some(_) => buildRecursively(list.tail, result + list.head)
+        case None => result
+      }
+    }
+
+    buildRecursively(list)
+  }
+
+  def getLongNum(list: List[String], words: List[String]): String = {
+    val therest = list.drop(list.length - 8)
+    val printable = list.take(list.length - 8).mkString("") +:
```

Comment:
mkString("") could be replaced with just mkString because "" is the default value